src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  }
```

Comment:
You are using `if else` structure here. It would be better to use `if` with `return` to simplify the code and improve readability. For example, you could write `if (days >= LONG_TERM_RENTAL_DAYS) { return days * DAILY_RENTAL_COST - LONG_TERM_DISCOUNT; }` and then `if (days >= SHORT_TERM_RENTAL_DAYS) { return days * DAILY_RENTAL_COST - SHORT_TERM_DISCOUNT; }`. This way, you avoid unnecessary `else` statement.